_posts/2021-03-23-keccak-optimizer-bug.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+published: true
+title:  "Solidity Keccak Caching Bug"
```

Comment:
Maybe add the word "optimizer" somewhere?

Reply:
```suggestion
title:  "Solidity Optimizer Keccak Caching Bug"
```

like this?

Reply:
@franzihei the suggestion was added.